.github/CODEOWNERS

```diff
@@ -8,71 +8,69 @@
 * @williamfalcon @borda @tchaton @awaelchli @kaushikb11 @rohitgr7
 
 # CI/CD and configs
-/.github/       @borda @carmocca @akihironitta @tchaton
-/.circleci/     @borda @carmocca @akihironitta @tchaton
-/.azure/        @borda @carmocca @akihironitta @tchaton
-/dockers/       @borda @carmocca @akihironitta @tchaton
-*.yml           @borda @carmocca @akihironitta @tchaton
+/.github/       @borda @carmocca @akihironitta @otaj
+/.circleci/     @borda @carmocca @akihironitta @otaj
+/.azure/        @borda @carmocca @akihironitta @otaj
+/dockers/       @borda @carmocca @akihironitta @otaj
+*.yml           @borda @carmocca @akihironitta @otaj
 
 # Docs
-/docs/                                      @edenlightning @tchaton @borda @awaelchli @RobertLaurella @Felonious-Spellfire
+/docs/                                      @edenlightning @tchaton @borda @awaelchli @RobertLaurella @rohitgr7
 /.github/*.md                               @edenlightning @williamfalcon @borda
-/.github/ISSUE_TEMPLATE/                    @edenlightning @borda @tchaton
+/.github/ISSUE_TEMPLATE/                    @edenlightning @borda @tchaton @awaelchli
 /docs/source-pytorch/conf.py                @borda @awaelchli @carmocca
 /docs/source-pytorch/index.rst              @williamfalcon
 /docs/source-pytorch/levels                 @williamfalcon @RobertLaurella
 /docs/source-pytorch/expertise_levels       @williamfalcon @RobertLaurella
-/docs/source-app/                           @williamfalcon @Felonious-Spellfire @RobertLaurella @tchaton @awaelchli
+/docs/source-app/                           @williamfalcon @RobertLaurella @tchaton @awaelchli
 /docs/source-app/conf.py                    @borda @awaelchli @carmocca
 /docs/source-app/index.rst                  @williamfalcon
-/docs/source-app/expertise_levels           @williamfalcon @Felonious-Spellfire @RobertLaurella
+/docs/source-app/expertise_levels           @williamfalcon @RobertLaurella
 
-# Packages
-/src/pytorch_lightning                      @borda @awaelchli @carmocca @justusschock @rohitgr7 @otaj
-/src/pytorch_lightning/accelerators         @williamfalcon @tchaton @SeanNaren @awaelchli @justusschock @kaushikb11
-/src/pytorch_lightning/callbacks            @williamfalcon @tchaton @carmocca @kaushikb11
-/src/pytorch_lightning/core                 @tchaton @borda @carmocca @justusschock @kaushikb11
```

Comment:
Generally dropped @tchaton and @kaushikb11 since they don't have much time to review now.